### PR TITLE
Updated docs to mention InfoDump for creating README.md, etc.

### LINKED ISSUE #3886: Info.lua documentation lacks information on how to generate README's
The [documentation](https://api.cuberite.org/InfoFile.html) on the Info.lua file does not specify how to generate README files for BB-code or Markdown. This is a major part when moving over to the `Info.lua` file and thus should be explained.

### LINKED ISSUE #3887: Info.lua documentation lacks information about command categories
The [documentation](https://api.cuberite.org/InfoFile.html) on Info.lua does not mention command categories at all. This can leave people wondering why InfoDump creates a "General" subheading in the commands section.

### PR DESCRIPTION
**Pull request by williamhatcher**
Added myself to CONTRIBUTORS. 
Should fix and close #3886 
Should fix and close #3887

### CODE REVIEW

**Comment by williamhatcher** [comment]
This can also close #3887

**Comment by williamhatcher** [comment]
This should also fix the broken forum styling in form_info.txt that InfoDump creates.

File: Server/Plugins/APIDump/InfoFile.html [hunk 1]
```diff
@@ -150,6 +156,8 @@
 
 		<p>The permission element specifies that the command is only available with the specified permission. Note that the permission for subcommand's parent isn't checked when the subcommand is called. This means that specifying the permission for a command that has subcommands has no effect whatsoever, but is discouraged because we may add processing for that in the future.</p>
 
+		<p>The Category string is only used for generating the documentation. Specifying different categories will result in commands grouped together, not specifying a category will place that command in the "General" category. Categories are optional.</p>
```
Comment (NiLSPACE) [inline Server/Plugins/APIDump/InfoFile.html]: Perhaps you can `<a href="#Categories">link this to the Categories section</a>`?

File: Server/Plugins/APIDump/InfoFile.html [hunk 2]
```diff
@@ -217,6 +225,24 @@
 
 
 			<hr />
+			<a name="Categories"><h2>Categories</h2></a>
+
+			<p>The optional Categories table provides descriptions for categories in the generated documentation. Commands can have categories with or without category descriptions in this table. The documentation generator will output a table of listed categories along with their description.</p>
```
Comment (madmaxoft) [inline Server/Plugins/APIDump/InfoFile.html]: This is a somewhat twisted description, I had to read it three times to understand it (and I know how all this works).

How about: 
```
The optional Categories table provides descriptions for command categories in the generated
documentation. The documentation generator will group the commands by their specified
Category ("General" by default) and each category will have the specified description
written to it.
```

File: Server/Plugins/APIDump/InfoFile.html (hunk 1 above)
Comment (williamhatcher) [inline Server/Plugins/APIDump/InfoFile.html]: Check line 23

File: Server/Plugins/APIDump/InfoFile.html (hunk 2 above)
Comment (williamhatcher) [inline Server/Plugins/APIDump/InfoFile.html]: I completely agree. I've made a few more commits, one of which uses your paragraph.

**Comment by williamhatcher** [comment]
Do I need to do anything else before this is merged, @madmaxoft?

**Comment by madmaxoft** [comment]
I already approved this, but we usually take at least two devs to look at the code before merging, so right now this waits for someone else to review it.

**Comment by williamhatcher** [comment]
Should I make any more changes to this branch before it gets merged?

**Pull request merged**